src/Strategy/Elixir/MixTree.hs

```diff
@@ -338,29 +356,27 @@ parseVerConstraint = do
     WildcardAny -> pure $ CEq "*"
   where
     findVersionText :: Parser Text
-    findVersionText = T.pack <$> some (alphaNumChar <|> char '.' <|> char '-' <|> char '*' <|> char '+')
+    findVersionText = Text.pack <$> some (alphaNumChar <|> char '.' <|> char '-' <|> char '*' <|> char '+')
 
--- | Parse mix constraint operator
-parseConstraintOperator :: Parser MixConstraintOperators
-parseConstraintOperator = fromMaybe Equal <$> optional (asum (map symbol operatorList) >>= textToMixVersion)
-  where
-    operatorList = [">=", "<=", ">", "<", "==", "!=", "~>", "="] :: [Text]
-
--- | Constructs `MixConstraintOperators` from textual representation (==, >=, etc.) of operators.
-textToMixVersion :: (MonadFail m) => Text -> m MixConstraintOperators
-textToMixVersion = \case
-  "==" -> pure Equal
-  "!=" -> pure NotEqual
-  ">=" -> pure GreaterThanOrEqual
-  "<=" -> pure LessThanOrEqual
-  ">" -> pure GreaterThan
-  "<" -> pure LessThan
-  "~>" -> pure Compatible
-  "*" -> pure WildcardAny
-  "=" -> pure Equal
-  other -> fail ("Could not recognize mix constraint operator: (" <> toString other <> ")")
-
--- | Parses [mix constraint expression](https://hexdocs.pm/elixir/1.12/Version.html)
+    parseConstraintOperator :: Parser MixConstraintOperators
+    parseConstraintOperator = fromMaybe Equal <$> optional (asum (map symbol operatorList) >>= textToMixVersion)
+      where
+        operatorList = [">=", "<=", ">", "<", "==", "!=", "~>", "="] :: [Text]
+
+    textToMixVersion :: (MonadFail m) => Text -> m MixConstraintOperators
+    textToMixVersion = \case
+      "==" -> pure Equal
+      "!=" -> pure NotEqual
+      ">=" -> pure GreaterThanOrEqual
+      "<=" -> pure LessThanOrEqual
+      ">" -> pure GreaterThan
+      "<" -> pure LessThan
+      "~>" -> pure Compatible
+      "*" -> pure WildcardAny
+      "=" -> pure Equal
+      other -> fail ("Could not recognize mix constraint operator: (" <> Text.unpack other <> ")")
```

Comment:
[nit]: prefer `toString` from `Data.String.Conversion` over `Text.unpack`
```suggestion
      other -> fail ("Could not recognize mix constraint operator: (" <> toString other <> ")")
```

Reply:
I've added this suggestion to style guide now - https://github.com/fossas/team-analysis/issues/671